ui/app/components/app-item/deployment.hbs

```diff
@@ -26,7 +26,7 @@
       <:badge>
         {{#if (eq @deployment.state 4)}}
           <b data-test-destroyed-badge class="badge badge--destroyed">
-            <Pds::Icon @type="trash" class="icon" />&nbsp; {{t "page.deployments.destroyed_label"}}
+            <FlightIcon @name="trash" class="icon" />&nbsp; {{t "page.deployments.destroyed_label"}}
```

Comment:
This seems a little tight in its box now. Maybe let’s add a touch more padding?

<img width="117" alt="CleanShot 2021-12-09 at 10 18 49@2x" src="https://user-images.githubusercontent.com/34030/145378136-b524e636-946b-4f60-b31e-e2c810d847ad.png">

Reply:
This is eventually going to be removed because the new designs that Al made for the deployment card states (in the redesign Figma). Should I just remove it in this PR?

Reply:
Ah — I’d say leave it as is. You’d probably have to change some tests too if you remove it.